src/index.ts

```diff
@@ -16,6 +16,7 @@
  * @public
  */
 export * from './functions';
+export * from './server';
```

Comment:
This lack of re-export is intentional.

The usage should be:

```ts
import {getTestServer} from '@google-cloud/functions-framework/testing';
```

Or, if you are trying to get `server`, you can import from `/server`.

See docs:

https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/docs/testing-functions.md

Reply:
> Or, if you are trying to get server, you can import from /server.

Thanks, though server isn’t currently included in the package.json exports

Reply:
Yes. Why do you want to import the internal server vs the test server?

Perhaps a different approach can be taken, since we can't export our internal server.

We have examples of integration tests.

Reply:
See my follow up comment here. 

https://github.com/GoogleCloudPlatform/functions-framework-nodejs/pull/463#issuecomment-1147332053

I don’t think I need this anymore but think the docs could be updated to show how you can test a function without it existing on disk.